tests/pax/nightly/common.libsonnet

```diff
@@ -0,0 +1,61 @@
+local common = import '../common.libsonnet';
+
+{
+  NightlyPaxTest:: common.PaxTest {
+    local config = self,
+    expPath:: '',
+    extraFlags:: '',
+    buildDate:: '$(date +%Y%m%d)',
+
+    // PAX tests are structured as bash scripts that run directly on the Cloud
+    // TPU VM instead of using docker images
+    testScript:: |||
+      set -x
+      set -u
+      set -e
+
+      # .bash_logout sometimes causes a spurious bad exit code, remove it.
+      rm .bash_logout
+
+
+      # check for nightly build
+      gsutil cp gs://pax-on-cloud-tpu-project/wheels/%(buildDate)s/paxml*.whl .
+      gsutil cp gs://pax-on-cloud-tpu-project/wheels/%(buildDate)s/praxis*.whl .
+
+      if [ -f praxis*.whl -a -f paxml*.whl ]; then
+          echo "Nightly builds succeeded."
+      else 
+          echo "Nighlty builds failed or are pending."
+          exit 1
+      fi 
+
+      # install pax and dependencies
+      pip install -U pip
+      pip install praxis*.whl
+      pip install paxml*.whl
+      sudo pip uninstall --yes jax jaxlib libtpu-nightly libtpu libtpu_tpuv4 tensorflow
+      git clone https://github.com/google/jax.git
+      cd jax/
+      pip install .
+      cd ..
+      pip install --pre -U jaxlib -f https://storage.googleapis.com/jax-releases/jaxlib_nightly_releases.html
+      pip install -U libtpu-nightly -f https://storage.googleapis.com/jax-releases/libtpu_releases.html
```

Comment:
Does the nightly `jaxlib` target the nightly `libtpu`? It looks like you overwrite `libtpu.so` below. Can you just install the pinned libtpu version with `pip`?

Reply:
yes the `jaxlib` has dependency on `libtpu`. At the moment, the available `libtpu` latest version is outdated, they are working on releasing the latest `libtpu`, so for now we are using a manually built version of `libtpu`. That's why I think we should clarify that this is temporary and we don't need both in the future

Reply:
yeah good catch, we want latest libtpu.so, which is built at head at gs://pax-on-cloud-tpu-project/libtpu/libtpu.so

Reply:
actually latest nightly libtpu worked, I modified the script to install that and not overwrite it